hydra/utils.py

```diff
@@ -37,7 +37,7 @@ def call(config: Union[ObjectConf, DictConfig], *args: Any, **kwargs: Any) -> An
             assert callable(type_or_callable)
             return _call_callable(type_or_callable, config, *args, **kwargs)
     except Exception as e:
-        raise InstantiateErrorException(f"Error calling '{cls}'") from e
+        raise CompactHydraException(f"Error calling '{cls}'") from e
```

Comment:
No, it's fine to have a special kind of exception for Instantiation errors (although InstantiationErrorException is a bit much, `InstantiateException` ?

The name of the exception gives the user more context about what went wrong.
also, a dedicated exception class can contain additional relevant fields like the cls in this case.

Reply:
+1 for renaming. Regrading, `dedicated exception class can contain additional relevant fields like the cls in this case.`, since I do not see use of this extra field right now, I am skipping it for now